generate/syntax/main.go

```diff
@@ -32,17 +32,18 @@ const (
 	destinationFileBeginning = "# Syntax" + newLine + "Below you will find the step syntax next to the name of the method it utilizes. Here GK stands for [Gherkin](https://cucumber.io/docs/gherkin/reference/#keywords) Keyword and words in brackets ([]) are optional:" + newLine
 )
 
-type replacee string
```

Comment:
typo: replacee

Reply:
It is meant to describe the subject that is going to be replaced by the `replacer`, hence `replacee`.